addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -170,8 +69,8 @@
 			<control type="label" id="40000">
 				<centerleft>50%</centerleft>
 				<centertop>175</centertop>
-				<width>50%</width>
-				<height>50</height>
+				<width>380</width>
+				<height>60</height>
```

Comment:
Just curious but why change seeklabel?

Reply:
To prevent overlap with the flags

Reply:
![image](https://github.com/user-attachments/assets/e5a810bb-dfa8-413c-9a6a-4253768e7088)